adding legacy template api and re-numbering the index API overall (#10330)
opensearch-project/documentation-website: _api-reference/nodes-apis/nodes-info.md

**Introduced 1.0**
{: .label .label-purple }

> The Nodes Info API represents mostly static information about your cluster's nodes, including but not limited to:

Reviewer: ```suggestion
The Nodes Info API represents mostly static information about your cluster's nodes, including the following:
```
